### PR TITLE
privacy policy may 2020 update

### PR DESCRIPTION
**Pull request by dvdokkum**

### CODE REVIEW

File: privacy_policy.md [hunk 1]
```diff
@@ -1,95 +1,180 @@
...
 
-1. Any public sector entity (or any entities providing services to suchentities) for surveillance purposes, including but not limited to:
-* Investigating or tracking LVN users or their LVN Content; and,
-* Tracking, alerting, or other monitoring of sensitive events (including but not limited to protests, rallies, or community organizing meetings);
+5. Any entity that you reasonably believe will use such data to violate the [Universal Declaration of Human Rights](http://www.un.org/en/documents/udhr/), including without limitation Articles 12, 18, or 19. 
 
-2. Any public sector entity (or any entities providing services to such entities) whose primary function or mission includes conducting surveillance or gathering intelligence;
+California Resident Rights
```
Comment (lilaconlee) [inline privacy_policy.md:132]: would we respond to similar requests from non-california residents?

File: terms_of_use.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+What about messaging?
+--------------------
+
+As part of the Services, you may receive communications through the Services, including messages that Cortico sends you (for example, via email or SMS). When signing up for the Services, you will receive a welcome message and instructions on how to stop receiving messages. **By signing up for the Services and providing us with your wireless number, you confirm that you want Cortico to send you information that we think may be of interest to you, which may include Cortico using automated dialing technology to text you at the wireless number you provided, and you agree to receive communications from Cortico, and you represent and warrant that each person you register for the Services or for whom you provide a wireless phone number has consented to receive communications from Cortico**. You agree to indemnify and hold Cortico harmless from and against any and all claims, liabilities, damages (actual and consequential), losses and expenses (including attorneys’ fees) arising from or in any way related to your breach of the foregoing.
```
Comment (lilaconlee) [inline terms_of_use.md:54]: would it be worth including anything about the user's ability to stop receiving messaging? maybe not the right place or not relevant yet.

File: terms_of_use.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+Are there restrictions in how I can use the Services?
+---------------------------------------------------- 
+
+1. You represent, warrant, and agree that you will not contribute any Content or User Submission (each of those terms is defined below) or otherwise use the Services or interact with the Services in a manner that:
```
Comment (lilaconlee) [inline terms_of_use.md]: should this part be enumerated?

**Review by lilaconlee — COMMENTED** [review]
thanks for pr-ing!

File: privacy_policy.md (hunk 1 above)
Comment (dvdokkum) [reply privacy_policy.md:132]: Yes. We'll add explicit coverage for states in the policy as they roll out or adopt their own privacy laws, but in practice we'll provide the same level of care/response for all US residents.

File: terms_of_use.md (hunk 2 above)
Comment (dvdokkum) [reply terms_of_use.md:54]: I think from a policy perspective this plus later language around closing an account covers the ability to stop messaging. But, this is a good reminder that we do need to add an explicit messaging opt-in to the registration flow.

File: terms_of_use.md (hunk 3 above)
Comment (dvdokkum) [reply terms_of_use.md]: It was originally enumerated with letters, not numbers... but I couldn't figure out how to do this in markdown so I switched it. Do you know of a way to do that? I guess I could just do it manually?

Comment (lilaconlee) [reply terms_of_use.md]: Yeah either manually or use HTML:
```html
<ol type="A"> 
<li>Coffee</li> 
<li>Tea</li> 
<li>Milk</li>
</ol>
```

**Pull request merged**